instrumentation/opentelemetry-instrumentation-pymongo/tests/test_pymongo.py

```diff
@@ -66,17 +71,24 @@ def test_started(self):
             span.attributes[SpanAttributes.NET_PEER_NAME], "test.com"
         )
         self.assertEqual(span.attributes[SpanAttributes.NET_PEER_PORT], "1234")
+        self.start_callback.assert_called_once_with(span, mock_event)
 
     def test_succeeded(self):
         mock_event = MockEvent({})
-        command_tracer = CommandTracer(self.tracer)
+        command_tracer = CommandTracer(
```

Comment:
Can we validate the the query was indeed captured? Same for the query result

Reply:
the query is captured by default

In these tests I just validate that the hooks are being called. 
In the SDK tests I'll validate we get the required data from them

Reply:
👍